### ISSUE TITLE
OSD Timer - option to switch between "total time" and "armed time"

### ISSUE THREAD

**Issue by NotFastEnuf** [NotFastEnuf/NFE_QUICKSILVER#4]
This has been requested a few times ....  making the changes to the osd timer is easily done, but some thought and discussion should be put into how and where the "switch" should be placed in both OSD and GUI.  Once this is discussed and decided - this feature can be added

**Comment by chaotixtech** [comment]
Able to change in both would be best but osd seems more important as it's mostly racers that like timer from plug in. 
Default should be on arm as most users expect flight time to show.

**Comment by NotFastEnuf** [comment]
Maybe we just add a second independent element instead of logic and a switch

**Comment by chaotixtech** [comment]
That sounds like the best idea, I do think default should be flight/arm time though.

**Comment by ProjectMockingbird** [comment]
I agree with this direction and agree with Chaotix about default as well.
